binderhub-service/templates/docker-api-network-policy.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: binder-dind-network-policy
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: binderhub-service
```

Comment:
Let's include these additional labels in both these cases so we only affect pods from our deployment? https://github.com/2i2c-org/binderhub-service/blob/main/binderhub-service/templates/docker-api/daemonset.yaml#L11

Reply:
These labels are only applied for on the docker api pods because are managed by this helm chart. The build pods are created by `KubernetesBuildExecutor` and are given one label I believe, which is:
https://github.com/jupyterhub/binderhub/blob/7f8b6c3137a6f8e66e6c193ee81d32bcf0826a6e/binderhub/build.py#L358

Does this sound right to you?

Reply:
@GeorgianaElena yea that makes sense to me! We should fix that upstream later on, but we don't need to handle that here.